src/pattern-library/routes.js

```diff
@@ -225,8 +221,18 @@ const routes = [
   },
   { title: 'LinkButton', group: 'input' },
   { title: 'TextField', group: 'input' },
-  { title: 'Card', group: 'layout' },
-  { title: 'Panel', group: 'layout' },
+  {
+    title: 'Card',
+    group: 'layout',
+    component: CardPage,
+    route: '/layout-card',
+  },
+  {
+    title: 'Panel',
+    group: 'layout',
+    component: PanelPage,
+    route: '/layout-panel',
+  },
```

Comment:
As with `ScrollBox` and `Scroll`, I think it would be good to put the high-level component first, or in some other way make the composite component, that is a recommended composition of the lower-level components, more discoverable.

Reply:
I've swapped this navigation ordering for now. `Card` vs. `Panel` is different than `ScrollBox` in that using `Card` directly is a common pattern, whereas using the "lower-level" `Scroll` components directly would be more uncommon.